release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -733,14 +728,16 @@ The kubelet is already publishing a set of network-observable related metrics. T
 
+See xref:../networking/associating-secondary-interfaces-metrics-to-network-attachments.adoc#associating-secondary-interfaces-metrics-to-network-attachments[Associating secondary interfaces metrics to network attachments] for more information.
+
 [id="ocp-4-6-cnf-tests-discovery-mode"]
 ==== CNF tests can be run in discovery mode
 
 There is an optional mode where the Cloud-native Network Functions (CNF) tests try to look for configurations on the cluster instead of applying the new ones. The CNF tests image is a containerized version of the CNF conformance test suite. It is intended to be run against a CNF-enabled {product-title} cluster where all the components required for running CNF workloads are installed.
 
 The tests must perform an environment configuration every time they are executed. This involves items such as creating SR-IOV Node Policies, Performance Profiles, or PtpProfiles. Allowing the tests to configure an already configured cluster might affect the functionality of the cluster. Also, changes to configuration items such as SR-IOV Node Policy might result in the environment being temporarily unavailable until the configuration change is processed.
 
-Discovery mode validates the functionality of a cluster without altering its configuration. Existing environment configurations are used for the tests. The tests attempt to find the configuration items needed and use those items to execute the tests. If resources needed to run a specific test are not found, the test is skipped, providing an appropriate message to the user. After the tests are finished, no cleanup of the pre-configured configuration items is done, and the test environment can be used immediately for another test run.
+xref:../scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc#discovery-mode[Discovery mode] validates the functionality of a cluster without altering its configuration. Existing environment configurations are used for the tests. The tests attempt to find the configuration items needed and use those items to execute the tests. If resources needed to run a specific test are not found, the test is skipped, providing an appropriate message to the user. After the tests are finished, no cleanup of the pre-configured configuration items is done, and the test environment can be used immediately for another test run.
```

Comment:
This one links to the top of the page - might need the "_cnf-performance-addon-operator-for-low-latency-nodes" added to the xref